bin/cron/deploy_to_test

```diff
@@ -6,28 +6,24 @@ require 'cdo/developers_topic'
 require 'cdo/github'
 require 'cdo/only_one'
 
-DEVELOPERS = 'developers'.freeze
 TOPIC_DTT_IN_PROGRESS = 'no (robo-DTT in progress)'.freeze
 TOPIC_DTT_FAILED = 'robo-DTT failed'.freeze
 
-def main
-  unless DevelopersTopic.dtt?
-    ChatClient.message(
-      'infra-test',
-      'robo-DTT skipped (DTT was not yes)',
-      color: 'yellow'
-    )
-    return
-  end
+def early_exit_if_appropriate
+  no_branches = {DTP: DevelopersTopic.dtp?, DTT: DevelopersTopic.dtt?, DTS: DevelopersTopic.dts?}.
```

Comment:
Hmm...true. But I also think we tend to keep `test` clean during HOC. For now, I'm going to keep things as they are, as being conservative by default doesn't seem bad.